### PR TITLE
WIP: add Astropy Learn and testing coordinator roles

### PR DESCRIPTION
**Pull request by taldcroft**
@kelle - please have a look.  We need some text describing the roles, could you help me out?  You can just put it here as a comment if that's easiest.

### CODE REVIEW

**Comment by eteq** [comment]
Some text for the "Astropy testing coordinator":

Lead development and maintenance of the testing infrastructure for Astropy and the helpers, including:
* Managing issues/pull request for the Astropy core package regarding testing infrastructure
* Managing issues/pull requests in the repositories containing the testing plugins, and determining when new plugins are required
* Maintaining the "metapackage" with the testing machinery (`pytest-astropy` at the time of this writing)
* Supporting and enabling affiliated package usage of the testing infrastructure


Also, this *might* be better called "Astropy testing maintainer".  I think of "coordinator" as more of a "people" role and "maintainer" as more of a "coding" role, and I think this is a bit more of the latter.  But I'm ok with it either way.

File: roles.txt [hunk 1]
```diff
@@ -18,8 +18,16 @@ Community engagement coordinator | Overall  | Kelle Cruz | UNFILLED
                                  | Facebook | Kelle Cruz | Tom Robitaille, Erik Tollerud
                                  | Conferences | Adrian Price-Whelan | Erik Tollerud, Tom Robitaille, Kelle Cruz
 
+Learn | Coordinator | Kelle Cruz | UNFILLED
+      | Slack | Tom Robitaille | UNFILLED
```
Comment (kelle) [inline roles.txt]: this line (Slack) goes above as a Community Engagement subrole.

**Comment by kelle** [comment]
Documentation & Examples Infrastructure Maintainer
Maintain the documentation website (http://docs.astropy.org/en/stable/index.html), including
- Managing the Sphinx infrastructure
- Implementing changes and improvements to the documentation website

Tutorial and Guides Infrastructure Maintainer 
Maintain the Tutorials website (http://www.astropy.org/astropy-tutorials/), including
- Facilitating the display and discoverability of the tutorials
- Rendering of the Jupyter notebooks
- Integrated testing of notebooks

Tutorials and Guides Content Coordinator
Oversee the material included in Tutorials and Guides, including:
- Reviewing issues and pull requests 
- Soliciting new content as needed
- Working with Infrastructure Maintainers to maintain website

**Comment by kelle** [comment]
ping on this.

**Comment by taldcroft** [comment]
@kelle - still need a role description for `Learn`.

Also, there is a small issue for Tutorials and Guides because the current infrastructure only supports a description for a role, but not the two sub-roles.  So I did something that roughly works:
image

There is probably a better way, so let me know.  It's somewhat annoying that the astropy.org template overrides the `<em>` tag so it does nothing, since I think that italics for the sub-role might work better than bold.  Also, `<h4>` gives exactly the same font as normal text, sigh.

p.s. humans (at least this human) were not designed for hand-editing HTML.  There must be a better way!

**Comment by eteq** [comment]
@taldcroft - I think what you did there with the infrastructure/content is fine.  We might want to re-think that formatting later (maybe fixing #194) but best to get the roles in ASAP since these folks are already doing the work.

Also, I think there was a miscommunication about the Slack role - that's community engagement *not* a "learn" role - the "learn" role is only a coordinator. I fixed that myself in 3a7b706 since it's pretty simple.

On the actual learn description, I think it's pretty straightforward and just pushed up a commit with a description that I think reflects the last discussion we had. @kelle, if you like what I put in this commit (7bb1e71), I think this PR is now good to go.

**Comment by kelle** [comment]
This looks great to me!

**Comment by eteq** [comment]
I'll merge this at the end of today unless @taldcroft has any objections.

**Pull request merged**